source/gloperate-glkernel/include/gloperate-glkernel/HemisphereDistributionKernelStage.h

```diff
@@ -71,7 +70,7 @@ class GLOPERATE_GLKERNEL_API HemisphereDistributionKernelStage : public gloperat
 
 protected:
     // Virtual Stage interface
-    virtual void onContextInit(gloperate::AbstractGLContext * context) override;
+    virtual void onContextInit(gloperate::AbstractGLContext *) override;
```

Comment:
Please restore the parameter names in the header files. For the definition of a function the parameter names may get omitted.